### DISCUSSION TITLE
I edited the Wiki page but you might want to host the image

### DISCUSSION THREAD

**Discussion by blundell** [nathanmarlor/foxess_modbus#402] [Ideas]
I edited this page: https://github.com/nathanmarlor/foxess_modbus/wiki/Waveshare-RS485-to-ETH-(B) and added a new image.

I was going to make a pull request and add it to the assets folder, like I saw other images in your wiki.
But I can't see that folder, so thought I'd make this post: you may want to host this image yourself.

This one:

<img src="https://lh3.googleusercontent.com/pw/AIL4fc8wEGZH6xV3OAoabjY5hLl_9z0oyOYd9pBKk3gcgysXF88Pn78emsz9teRJoXgf8vQOShCiSZdzC8fIfshtScFoNimmkxAbnoEL8M2-zFt0c1Gm-k_ZAZ2_3ktz-Ymk_FI5PlQpDDsZAxJSmJZHGc26kg=w481-h759-s-no" width=50%/>

**Comment by canton7** [comment]
Thanks!

It's weird that the "add image" button on the wiki editor doesn't offer to let you upload an image, but if you copy the image to your clipboard and then paste it into the wiki editor, it gets uploaded.

**Comment by blundell** [comment]
Ah thanks, I was thinking you'd pushed the images to the repo to store them.
Yeah I was caught out by the dialog only taking a link, good old copy an paste - I've done it 👍 thanks!

**Reply by canton7** [reply]
Fab, thank you!
